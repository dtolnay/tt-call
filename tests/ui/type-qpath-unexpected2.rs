#[macro_use]
extern crate tt_call;

tt_call! {
    macro = [{ parse_type }]
    input = [{ <T as @ }]
}

fn main() {}
