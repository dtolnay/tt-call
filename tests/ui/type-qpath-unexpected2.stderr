error: no rules expected the token `@`
 --> $DIR/type-qpath-unexpected2.rs:5:22
  |
5 |     input = [{ <T as @ }]
  |                      ^ no rules expected this token in macro call
